pkg/webui/console/containers/notifications/index.js

```diff
@@ -55,13 +63,18 @@ const indicesToPage = (startIndex, stopIndex, limit) => {
 const m = defineMessages({
   seeArchived: 'See archived messages',
   seeAll: 'See all messages',
+  noNotificationsSubtitle: 'Once you receive a notification, it can be viewed here.',
```

Comment:
```suggestion
  noNotificationsSubtitle: 'Once you receive a notification, it can be viewed here',
```
Don't use end of sentence periods, unless the label/description/caption consists of more than one sentence.